File: euler-2.py
one = 1
second = 2
count = 2
new = True


for i in range(100):
    if(new):
        new = False
        one = one + second
        item = one
    else: 
        new = True
        second = one + second
        item = second

    if item % 2 == 0 and count < 4000000:
        count += item

    
print(count)

File: java/euler2.java
public class euler2 {
	public static void main(String[] args) {

        int one = 1;
        int second = 2;
        int count = 2;
        int item;
        boolean sItem = true;

        for (int i = 0; i < 100; i++) {
            if(sItem){
                sItem = false;
                one = one + second;
                item = one;
            }else{
                sItem = true;
                second = one + second;
                item = second;
            }

            if(item % 2 == 0 && count < 4000000){
                count += item;
            }
        }

        System.out.println(count);

	}
}
